### ISSUE TITLE
Still have this problem now, someone is solved?

### ISSUE THREAD

**Issue by trista00** [ariya/phantomjs#14875]
1. Which version of PhantomJS are you using? Tip: run `phantomjs --version`.

2. What steps will reproduce the problem?
  1.
  2.
  3.

3. Which operating system are you using?

4. Did you use binary PhantomJS or did you compile it from source?

5. Please provide any additional information below.

**Comment by trista00** [comment]
Rasterize.js renders black below a specific height when screen capture a long web page.

**Comment by trista00** [comment]
Friends.

I am new to phantomjs, I am using it to create screenshots of web pages with user created content. It works fine with most of the pages, but I just found it doesn't work well with a quite long web page. The lower part of the image created by phantomjs is totally black.

I changed several parameters like page.viewPort.height / page.clipRect.height, but every time the black part starts at the same place, so I am curious why phantomjs can't render correctly below a certain height? Are there any parameters I need to tweak accordingly?
